.npmignore

```diff
@@ -0,0 +1,2 @@
+.github
```

Comment:
```suggestion
.github/
```

Since `.github` is always a directory, we can add a slash after it.

Reply:
Agree. I tend to use `.github` because it works as a glob pattern, but explicit is probably better than implicit. Changed.